device_samples/getting_started/connections/connection_string.ts

```diff
@@ -10,45 +10,43 @@ import { Mqtt as Protocol } from 'azure-iot-device-mqtt';
 
 import { Client, Message } from 'azure-iot-device';
 
-// String containing Hostname, Device Id & Device Key in the following formats:
-//  "HostName=<iothub_host_name>;DeviceId=<device_id>;SharedAccessKey=<device_key>"
-const deviceConnectionString: string = process.env.IOTHUB_DEVICE_CONNECTION_STRING ?? '';
-const logRed: string = '\x1b[31m%s\x1b[0m';
+// String containing Hostname, Device ID, and Device Key in the following format:
+// "HostName=<IoT Hub hostname>;DeviceId=<device id>;SharedAccessKey=<device key>"
+const deviceConnectionString = process.env.IOTHUB_DEVICE_CONNECTION_STRING || '';
+const logRed = '\x1b[31m%s\x1b[0m';
```

Comment:
*Perhaps* more clear...
Styling console output
You can use the %c directive to apply a CSS style to console output:

console.log("This is %cMy stylish message", "color: yellow; font-style: italic; background-color: blue;padding: 2px");
The text before the directive will not be affected, but the text after the directive will be styled using the CSS declarations in the parameter.

You may use %c multiple times:
console.log("Multiple styles: %cred %corange", "color: red", "color: orange", "Additional unformatted message");

Reply:
This does not seem to work for me, using Node 16

Reply:
But I read it on the internet!